scripts/pre_commit_linter.py

```diff
@@ -44,12 +44,11 @@
 
 __author__ = "Barnabas Makonda(barnabasmakonda@gmail.com)"
 
+import sys
+import time
 import argparse
 import os
 import subprocess
-import sys
-import time
```

Comment:
Definitely, errors (from running the pylint on this file--pre_commit_linter.py):
- C0413 - wrong-import-position
- W0612 - used-variable (this should **not** be ignored)
- R0914 - too-many-locals
- W0141 - bad-builtin
- C0411 - wrong-import-order

Here is the dump: [pre_commit_linter_errors.txt](https://github.com/oppia/oppia/files/75458/pre_commit_linter_errors.txt) for more context.